code/modules/dwarven/dwarven_items.dm

```diff
@@ -0,0 +1,244 @@
+/obj/item/book_of_babel/dwarven_guide
+	name = "Brokering 101: Dwarven guide"
+	desc = "An ancient tome written dwarven"
+	icon = 'icons/obj/library.dmi'
+	icon_state = "book1"
+	w_class = 2
+
+/obj/item/book_of_babel/dwarven_guide/attack_self(mob/living/carbon/human/user)
+	if(!user.has_language(/datum/language/dwarven))
+		return FALSE
+	. = ..()
+
+/obj/item/twohanded/war_hammer
+	name = "dwarven warhammer"
+	desc = "A very heavy warhammer, used to dent skulls unless they are already dented."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "greyscale_dwarven_warhammer0"
+	lefthand_file = 'icons/mob/inhands/weapons/hammers_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/hammers_righthand.dmi'
+	flags_1 = CONDUCT_1
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR
+	force = 14
+	force_unwielded = 14
+	force_wielded = 20
+	armour_penetration = 20
+	w_class = WEIGHT_CLASS_BULKY
+	custom_materials = list(/datum/material/iron = 20000)
+	attack_verb = list("smashed", "dented", "bludeoned")
+	hitsound = 'sound/weapons/smash.ogg'
+	sharpness = IS_BLUNT
+
+
+/obj/item/twohanded/war_hammer/update_icon_state()
+	icon_state = "greyscale_dwarven_warhammer[wielded]"
+
+/obj/item/hatchet/dwarven
+	name = "dwarven "
+	desc = "Dwarf dwarf dwarf dwarf dwarf dwarf? DWARF!"
```

Comment:
best description you have in here

Reply:
You forgot to set the name to dwarven hatchet.
Dorf hits Penny Daniels with the dwarven !